conda/meta.yaml

```diff
@@ -11,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

Comment:
It is best to not allow `pip` to install missing build dependencies from PyPI (which is what `--no-build-isolation` does).

The quotes are not needed.